packages/gatsby-source-contentful/src/__tests__/__snapshots__/normalize.js.snap

```diff
@@ -2610,17 +2611,22 @@ Our Lemnos products are made carefully by our craftsmen finely honed skillful te
       ],
       "companyDescription___NODE": "JrePkDVYomE8AwcuCUyMicompanyDescriptionTextNode",
       "companyName___NODE": "JrePkDVYomE8AwcuCUyMicompanyNameTextNode",
+      "email": "",
       "id": "JrePkDVYomE8AwcuCUyMi",
       "internal": Object {
-        "contentDigest": "60153e88a39483fa024d4bf41ab95b6a",
+        "contentDigest": "98dc28795de1ada78c781c9979afb19b",
         "type": "ContentfulBrand",
       },
       "logo___NODE": "c4zj1ZOfHgQ8oqgaSKm4Qo2",
       "node_locale": "en-US",
       "parent": "Brand",
+      "phone": Array [
+        "",
+      ],
```

Comment:
Is it something we want or should I leave it like an empty array (please see my long comment above for details on why I changed that)

Reply:
Yeah we'd need this or else the schema would be wrong e.g. Gatsby wouldn't know that phone is an array of strings.